DigiThinkIT/erpnext
# fix: employee checkin permissions

Changes made:
- Employee Checkin DocType Log Type field made mandatory

Steps to test:
1. Create a checkin / checkout request
1. Leave log type field empty
1. Save request should be stopped
